spec/features/shelters/show_spec.rb

```diff
@@ -77,3 +77,51 @@
     end
   end
 end
+
+describe 'As a visitor' do
+  describe 'When I visit a shelters show page' do
+    it 'should show a list of reviews for that shelter' do
+      shelter1 = Shelter.create!(name: 'Dogs and Cats',
+                                 address: '1234 spoon.st',
+                                 city: 'Tampa',
+                                 state: 'Florida',
+                                 zip: '34638')
+      user1 = User.create!(name: 'Bob',
+                           street_address: '1234 Test Dr',
+                           city: 'Denver',
+                           state: 'Colorado',
+                           zip: '12345')
+      user2 = User.create!(name: 'Tony',
+                           street_address: '1234 Review Dr',
+                           city: 'Denver',
+                           state: 'Colorado',
+                           zip: '19345')
+      review1 = Review.create!(title: 'My Rating',
+                               rating: 3,
+                               content: 'The place is not bad.',
+                               picture: "https://upload.wikimedia.org/wikipedia/commons/6/6a/Bob_Gibson_crop.JPG",
+                               shelter_id: shelter1.id,
+                               user_id: user1.id,
+                               name: user1.name)
+      review2 = Review.create!(title: 'My thoughts',
+                               rating: 4,
+                               content: 'Great place for pets',
+                               picture: "https://upload.wikimedia.org/wikipedia/commons/4/4e/Tony_Romo_2015.jpg",
+                               shelter_id: shelter1.id,
+                               user_id: user2.id,
+                               name: user2.name)
+      visit "/shelters/#{shelter1.id}"
+
+      expect(page).to have_content(review1.title)
```

Comment:
Once an edge case shelter and review are added, then you can add tests like: 
`expect(page).to_not have_content(review2.rating).to_s`
etc.